Formula/o/openvino.rb

```diff
@@ -1,8 +1,8 @@
 class Openvino < Formula
   desc "Open Visual Inference And Optimization toolkit for AI inference"
   homepage "https://docs.openvino.ai"
-  url "https://github.com/openvinotoolkit/openvino/archive/refs/tags/2023.3.0.tar.gz"
-  sha256 "27cff20ac0662f5495d2c2eec47cbe5469ab2f225aa091d223f8bfc9d32f4fc3"
+  url "https://github.com/openvinotoolkit/openvino/archive/refs/tags/2024.0.0.tar.gz"
+  sha256 "b3c257f8af9545ae68a6ea217173b2b2de9dd42d35e8703a7a51d76f4c2bfe2f"
   license "Apache-2.0"
   head "https://github.com/openvinotoolkit/openvino.git", branch: "master"
 
```

Comment:
we need to add dependency to https://formulae.brew.sh/formula/python-packaging#default because of https://github.com/openvinotoolkit/openvino/commit/27e01f6d0f2280a1e99bc8fdb7acbcecda897937

Reply:
I've added 
` depends_on "python-packaging" => :build`
Could you please check?

Reply:
We're getting rid of the `python-*` formulae. Can this be a `resource` instead?